fladle-plugin/src/test/java/com/osacky/flank/gradle/integration/FulladlePluginIntegrationTest.kt

```diff
@@ -290,7 +300,7 @@ class FulladlePluginIntegrationTest {
            test: [0-9a-zA-Z\/_]*/android-project2/build/outputs/apk/androidTest/debug/android-project2-debug-androidTest.apk
            max-test-shards: 5
            environment-variables:
-               "clearPackageData": "true"
+               "clearPackageData": "false"
```

Comment:
A question. " usage in here looks wrong. Shouldnt be it just `clearPagkageData: false`

Reply:
I think you are right. Maybe these integration tests aren't working properly since this other PR has written it the other way: https://github.com/runningcode/fladle/pull/266/files#diff-8c950a3cebeff2c0a82c58117c197724232bb55f3341881c5715c27701437e90R117

Reply:
What is the actual flank.yml file look like?

Reply:
Integrations test are bit confusing. It might make sense to split cases and test case by case. I will try to find out actual flank.yml and check removing ""